frontend/views/containers/sidebar/Navigation.vue

```diff
@@ -43,8 +44,10 @@ nav.c-navigation(
             i18n Inbox (deprecated)
 
         .c-navigation-separator(v-if='groupsByName.length < 2')
-          router-link.button.is-small.is-outlined(
-            to='/new-group/name'
+          button(
+            class='is-small is-outlined'
+            @click='openModal("CreateGroup")'
+            data-test='createGroup'
```

Comment:
Down below, same issue (missing `:` in front of `:alt`), and might need to be `:aria-label`, not sure)